tests/graphical.rs

```diff
@@ -865,16 +905,18 @@ fn related_source_code_propagation() -> Result<(), MietteError> {
  3 │     here
    ╰────
   help: try doing it better next time?
-
-Error: oops::my::bad
-
-  × oops!
-   ╭─[bad_file.rs:1:1]
- 1 │ source
-   · ───┬──
-   ·    ╰── this bit here
- 2 │   text
-   ╰────
+╭─There were 1 related diagnostics:
```

Comment:
The vertical `|` works quite nicely to "mark" that the entire block is composed of related diagnostics, but it does feel like it crowds the left area: I would find it more difficult to quickly scan the output and determine how many errors I encountered.

I wonder if this could be helped by nesting the vertical line itself a bit more - e.g. 
```
  There were 1 related diagnostics:
  │ 
  ├─ 1.Error: oops::my::bad
  │ 
  │   × oops!
  │    ╭─[bad_file.rs:1:1]
  │  1 │ source
  │    · ───┬──
  │    ·    ╰── this bit here
  │  2 │   text
  │    ╰────
```